src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.container {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 96px;
+  height: 96px;
+}
+
+.stars-star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(n+1):nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(n+1):nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(n+1):nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(n+1):nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
you can join it with `,`

Reply:
How can i join it, when for every class i use different properties? And why all of tests are failed?